aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AemAnalyseMojo.java

```diff
@@ -207,12 +206,19 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         final VersionUtil versionUtil = new VersionUtil(this.getLog(), this.project, 
                 this.artifactHandlerManager, this.artifactMetadataSource, 
-                this.remoteArtifactRepositories, this.localRepository,
-                this.mavenSession.isOffline());
+                this.project.getRemoteArtifactRepositories(), this.project.getPluginArtifactRepositories(), 
+                this.mavenSession.getLocalRepository(), this.mavenSession.isOffline());
+
+        versionUtil.checkPluginVersion(this.plugin.getGroupId(), this.plugin.getArtifactId(), this.plugin.getVersion());
 
         final ArtifactId sdkId = versionUtil.getSDKArtifactId(this.sdkArtifactId, this.sdkVersion, this.useDependencyVersions);
         final List<ArtifactId> addons = versionUtil.discoverAddons(this.addons, this.useDependencyVersions);
 
+        // log warnings at start and end
```

Comment:
Maybe I'm missing it but doesn't this only print the warnings at the start (not the end?)

Reply:
Ah I see it now, `versionUtil.getVersionWarnings()` is called again later on.

Reply:
Yes, the warnings are printed later on to have a single set of warnings as a report